autoscheduler/frontend/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.urls import path
+from django.urls import path, re_path
 
 from . import views
 
 urlpatterns = [
+    # match the root
     path('', views.index),
+    # match all other pages
+    re_path(r'^(?:.*)/?$', views.index),
```

Comment:
I believe we could replace this with 
```
    # match all pages other than admin
    re_path(r'^(?!admin/).*', views.index),
```

Reply:
Won't we need `/api/` routes to be routed to `/scraper/`?

Reply:
This can be achieved with this line instead:
`re_path(r'^(?![(admin|api)]/).*', views.index),`
and add whatever other paths we don't want to match in the or

Reply:
I think that the `/api` oaths can be handled in `autoscheduler/urls.py`, the same way that `/admin` routes are handles currently

Reply:
Oh yeah you're totally right, didn't really think about that this was `frontend/views.py`.